src/api/ranking/ranking.types.ts

```diff
@@ -0,0 +1,3 @@
+export enum Goal {
+  GOAL = 2222,
```

Comment:
Trochę to sztuczne, nie uważasz? Te dane przyjdą z API, więc można to w ogóle wywalić